RMG/2Day/두 정수 사이의 합.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * a와 b를 입력받아 a~b사이의 모든값을 구하는 함수입니다.
+ * solution(3, 5) // 3 + 4 + 5 = 12
+ * solution(3, 3) // 3
+ * solution(5, 3) // 3 + 4 + 5 = 12
+ */
+function solution(a, b) {
+  let answer = 0;
+
+  if (a <= b) {
+    // a가 b보다 작거나 같을경우
+    for (let i = a; i <= b; i++) {
+      // for문을 사용하여 a입력값부터 b입력값을 answer에 저장합니다.
+      answer += i; //입력받은 값을 계속 더해줍니다.
+    }
+  } else {
+    // a가 b 보다 클경우
+    let x = b,
+      y = a; // a와 b의 값을 바꾸어서 for문을 실행합니다.
+    for (let i = x; i <= y; i++) {
+      answer += i;
+    }
```

Comment:
```suggestion
    for (let i = b; i <= a; i++) {
      answer += i;
    }
```
x, y 변수를 따로 할당 할 필요없이 for문 내에서만 변경해도 될 것 같은데, 따로 변수를 선언한 이유가 있을 까요?! 🤗

Reply:
단순히 a와 b의 자리를 바꾸어야만 한다고 생각하여 다시 변수를 선언하였는데, 말씀을 들어보니 for문에서 감소로 구성하는것이 더욱 깔끔했을 거 같습니다 !! 감사합니다!! 😊

Reply:
@Ryu-02 이해했습니다 :) 답변 감사합니다